packages/medusa/src/services/__tests__/product-category.ts

```diff
@@ -46,4 +46,73 @@ describe("ProductCategoryService", () => {
       )
     })
   })
+
+  describe("delete", () => {
+    const productCategoryRepository = MockRepository({
```

Comment:
suggestion(non blocking): Already mention in another pr, but all that can go to a fixture file, the container can inherit from the default, etc. Have a look to the payment provider test :D

Reply:
I've created is a ticket to handle this separately (CORE-977), will get to it as soon as the admin API endpoints are merged.